contrib/docker/php.production.ini

```diff
@@ -679,7 +679,7 @@ auto_globals_jit = On
 ; Its value may be 0 to disable the limit. It is ignored if POST data reading
 ; is disabled through enable_post_data_reading.
 ; http://php.net/post-max-size
-post_max_size = 64M
```

Comment:
TODO: want to make these variables that are interpolated during `docker entrypoint` (ideally derived from `.env` config file values from `MAX_PHOTO_SIZE * MAX_ALBUM_LENGTH + buffer`